show ProjectOverview in viewer mode and enable opening/running query builder in read only mode (#753)
finos/legend-studio: .changeset/tester-coffee-maker.md

---
'@finos/legend-studio-extension-query-builder': patch
---

> Enable opening query builder in read only via Mapping Test and Service. Disable saving query in read only mode.

Reviewer: ```suggestion
Enable opening query builder in read-only mode for mapping test and service.
```
